packages/website/default-pages/index.tsx

```diff
@@ -108,24 +108,18 @@ class HomePage extends React.Component {
                   <Heading>{context.siteName}</Heading>
                   <Section>
                     <PanelGrid displayAsColumn={displayAsColumn}>
-                      {Object.keys(pageInfo).find(x => x === 'readme') && (
-                        <Panel href="/readme" {...this.getReadmePanelProps()} />
+                      {context.readMe && (
+                        <Panel {...this.getReadmePanelProps(context.readMe)} />
                       )}
                       <Panel
-                        href="/packages"
-                        {...this.getPackagesPanelProps(
-                          context.packagesDescription,
-                        )}
+                        {...this.getPackagesPanelProps(context.packages)}
                       />
-                      {Object.keys(pageInfo)
-                        .slice(1)
-                        .filter(x => x !== 'readme')
-                        .map((key, i) => (
-                          <Panel
-                            key={key}
-                            {...this.getDocsPanelProps(context[i])}
-                          />
-                        ))}
+                      {context.docs.map(doc => (
```

Comment:
Just checking that this is never undefined, with no docs we return an empty array?

Reply:
Yep, it's handled in `processConfig`.
There's a `docsList.map()` that would blow up in `processConfig` before getting here.